flink-core/src/main/java/org/apache/flink/configuration/ExecutionOptions.java

```diff
@@ -55,4 +56,11 @@
 						"throughput")
 				)
 				.build());
+
+	public static final ConfigOption<List<String>> JOB_LISTENERS =
```

Comment:
I would suggest to move it to the `DeploymentOptions`.

Reply:
i have moved the argument from ExecutionOptions to   DeploymentOptions.